{"uuid": "e0522190-d1b8-42dc-bc62-a9f823875c6b", "children": ["0109afdf-8e21-4aa2-8a2b-5c4ed3f54e14"], "befores": [{"name": "browser", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: Browsing context has been discarded\n\n", "trace": "  File \"c:\\users\\patelvi\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\users\\patelvi\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 949, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\users\\patelvi\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 775, in call_fixture_func\n    res = next(it)\n  File \"C:\\Users\\patelvi\\PycharmProjects\\NCE\\onx\\test_multi.py\", line 47, in browser\n    driver.maximize_window()\n  File \"c:\\users\\patelvi\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 737, in maximize_window\n    self.execute(command, params)\n  File \"c:\\users\\patelvi\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\users\\patelvi\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1574177636827, "stop": 1574177637908}], "start": 1574177636827, "stop": 1574177638221}